pkg/api/types.go

```diff
@@ -221,6 +224,17 @@ type GCEPersistentDisk struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// A ISCSI Disk can only be mounted as read/write once.
+type ISCSIDisk struct {
```

Comment:
iSCSI is a lot less cloud-specific than GCE PD, so maybe the answer is yes, but...

Do we really want to expose this as a discrete type?  Maybe we want to glue this into @markturansky 's larger storage refactoring instead?  This is just another form of pet storage...